### PR TITLE
Update README according to today's meeting

### PR DESCRIPTION
**Pull request by macemoth**

### CODE REVIEW

**Review by jo9x — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -31,13 +12,12 @@ To add this library to your project, insert following to your `pom.xml`:
 | ------------------------------ | ------------------------------------ | ------------------------- |
 |                                | Auction ➡️ (over AuctionStartedEvent) | HTTP POST @ `/auction`    |
```
Comment (jo9x) [inline README.md]: At this place it would be good to note down that we handle this stuff via Puhubsubsub or MQTT

File: README.md [hunk 2]
```diff
@@ -90,29 +70,73 @@ To add this library to your project, insert following to your `pom.xml`:
...
\ No newline at end of file
+- `200` (Accepted) The patch was applied to the Task
+- `404` (Not Found) The Task requested to patch wasn't found
+- `406` (Not acceptable) An invalid patch request was made
+
+**Response Body**
+For debugging purposes, the patched Task is returned.
+    
+## Task Types (with in- and outputs)
+
```
Comment (jo9x) [inline README.md:123]: I think we differentiate between big and small robot?

File: README.md (hunk 1 above)
Comment (macemoth) [reply README.md]: Updated in 7793608

File: README.md (hunk 2 above)
Comment (macemoth) [reply README.md:123]: Updated in 7793608

**Pull request merged**